api/config.py

```diff
@@ -1,41 +0,0 @@
-import os
-
-import dotenv
-
-DEFAULTS = {
-}
-
-
-def get_env(key):
-    return os.environ.get(key, DEFAULTS.get(key))
-
-
-def get_bool_env(key):
-    value = get_env(key)
-    return value.lower() == 'true' if value is not None else False
-
-
-def get_cors_allow_origins(env, default):
-    cors_allow_origins = []
-    if get_env(env):
-        for origin in get_env(env).split(','):
-            cors_allow_origins.append(origin)
-    else:
-        cors_allow_origins = [default]
-
-    return cors_allow_origins
-
-
-class Config:
-    """Application configuration class."""
-
-    def __init__(self):
-        dotenv.load_dotenv()
-
-        self.TESTING = False
```

Comment:
It seems like the parameter TESTING is missing

Reply:
`TESTING` seems a legacy config and has no proper use.

Reply:
Moved `TESTING` into DeploymentConfig , as you requested.